packages/kotti-ui/source/kotti-table/KtTable.vue

```diff
@@ -322,8 +322,114 @@ export default {
 
 @import '../kotti-style/_variables.scss';
 
+table.kt-table {
+	position: relative;
+	width: 100%;
+	hyphens: auto;
+	table-layout: auto;
+	border-collapse: collapse;
+}
+
+::v-deep table.kt-table th {
```

Comment:
why do you need a v-deep for the `th` here?

Reply:
Because the th is rendered in other components/files and the styles here in KtTable are scoped, so without the v-deep, styles aren't applied